### PR TITLE
Change Batch nuget package name

### PR DESCRIPTION
**Pull request by xingwu1**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by xingwu1** [comment]
@shahabhijeet @matthchr  Could you review the code?

File: src/SDKs/Batch/Support/FileStaging/Batch.FileStaging/Batch.FileStaging.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <PackageId>Azure.Batch.FileStaging</PackageId>
+    <PackageId>Microsoft.Azure.Batch.FileStaging</PackageId>
```
Comment (dsgouda) [inline src/SDKs/Batch/Support/FileStaging/Batch.FileStaging/Batch.FileStaging.csproj]: @shahabhijeet do we need to publish a new package and publish backlinks to the old package here?

File: src/SDKs/Batch/DataPlane/changelog.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
-# Azure.Batch release notes
+# Microsoft.Azure.Batch release notes
+
+## Change in 8.1.1
+### Rename Nuget package name from Azure.Batch to Microsoft.Azure.Batch
+
+# Previous Azure.Batch release notes
```
Comment (matthchr) [inline src/SDKs/Batch/DataPlane/changelog.md]: Explain here:
"Prior to version 8.1.1, this package was named "Azure.Batch" on Nuget. The release notes below are for that package."

**Comment by xingwu1** [comment]
@shahabhijeet I will close this PR, and open a new PR to update the existing nuget package with updated metadata.

**Pull request closed without merging**